plugins/wasi_nn/CMakeLists.txt

```diff
@@ -149,6 +149,64 @@ if(BACKEND STREQUAL "ggml")
   endif()
 endif()
 
+if(BACKEND STREQUAL "neuralspeed")
+  find_package(simdjson QUIET)
```

Comment:
We changed the way to find the `simdjson` package, please use the new way like this pr: https://github.com/WasmEdge/WasmEdge/pull/3426

Reply:
I have changed. Please check, thank you.